<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogImageOptions</class>
 <widget class="QDialog" name="DialogImageOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>成像选项</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="coordinateOptions">
     <property name="title">
      <string>座标系</string>
     </property>
     <layout class="QFormLayout" name="formLayout_7">
      <item row="0" column="1">
       <widget class="QComboBox" name="coordinateType">
        <item>
         <property name="text">
          <string>平面</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>圆盘</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>机器人</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="coordinateTypeTitle">
        <property name="text">
         <string>座标类型</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="cyclularCenterTitle">
        <property name="text">
         <string>圆心位置</string>
        </property>
        <property name="buddy">
         <cstring>cyclularCenter</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="cyclularCenter">
        <property name="minimum">
         <double>-9999999.990000000223517</double>
        </property>
        <property name="maximum">
         <double>9999999.990000000223517</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QFormLayout" name="formLayout_5">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="scaleHTitle">
        <property name="text">
         <string>缩放</string>
        </property>
        <property name="buddy">
         <cstring>scaleH</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="scaleH">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>0.000100000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QFormLayout" name="formLayout_6">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="scaleVTitle">
        <property name="text">
         <string>缩放</string>
        </property>
        <property name="buddy">
         <cstring>scaleV</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="scaleV">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>0.000100000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QGroupBox" name="channelOffset">
     <property name="title">
      <string>通道偏移</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTreeWidget" name="channelOffsetList">
        <property name="editTriggers">
         <set>QAbstractItemView::DoubleClicked</set>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="indentation">
         <number>0</number>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
        <column>
         <property name="text">
          <string>通道</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>通道偏移</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="addOffset">
          <property name="text">
           <string>添加</string>
          </property>
          <property name="icon">
           <iconset resource="../../../Resources/Resources.qrc">
            <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="editOffset">
          <property name="text">
           <string>编辑</string>
          </property>
          <property name="icon">
           <iconset resource="../../resources/icons/Fatcow.qrc">
            <normaloff>:/fatcow/16x16/bullet_edit.png</normaloff>:/fatcow/16x16/bullet_edit.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="deleteOffset">
          <property name="text">
           <string>删除</string>
          </property>
          <property name="icon">
           <iconset resource="../../../Resources/Resources.qrc">
            <normaloff>:/icons/minus.png</normaloff>:/icons/minus.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>成像数据</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>通道</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="channels">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>归并</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="reduce">
        <item>
         <property name="text">
          <string>覆盖</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>取最大值</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>取最小值</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="directionFilterTitle">
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="toolTipDuration">
         <number>-1</number>
        </property>
        <property name="text">
         <string>方向</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="directionFilter">
        <item>
         <property name="text">
          <string>禁用</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>仅正向</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>仅反向</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="gateFeatureTitle">
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>归并参考特征值</string>
        </property>
        <property name="text">
         <string>特征值</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="gateFeature"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="optionsV">
     <property name="title">
      <string>垂直</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="axisVTitle">
        <property name="text">
         <string>成像轴</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="axisV"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="sourceFromVTitle">
        <property name="text">
         <string>起点</string>
        </property>
        <property name="buddy">
         <cstring>sourceFromV</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="sourceFromV">
        <property name="minimum">
         <double>-9999999.990000000223517</double>
        </property>
        <property name="maximum">
         <double>9999999.990000000223517</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="sourceToVTitle">
        <property name="text">
         <string>终点</string>
        </property>
        <property name="buddy">
         <cstring>sourceToV</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="sourceToV">
        <property name="minimum">
         <double>-9999999.990000000223517</double>
        </property>
        <property name="maximum">
         <double>9999999.990000000223517</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="sourceWidthVTitle">
        <property name="text">
         <string>宽度</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="sourceWidthV">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="gridVTitle">
        <property name="text">
         <string>网格尺寸</string>
        </property>
        <property name="buddy">
         <cstring>gridV</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="gridV">
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="pixelTitleV">
        <property name="text">
         <string>像素数</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="pixelV">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="backlashTitleV">
        <property name="text">
         <string>返程补偿</string>
        </property>
        <property name="buddy">
         <cstring>backlashV</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QDoubleSpinBox" name="backlashV">
        <property name="minimum">
         <double>-99.989999999999995</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="optionsH">
     <property name="title">
      <string>水平</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="axisHTitle">
        <property name="text">
         <string>成像轴</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="axisH"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="sourceFromHTitle">
        <property name="text">
         <string>起点</string>
        </property>
        <property name="buddy">
         <cstring>sourceFromH</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="sourceFromH">
        <property name="minimum">
         <double>-9999999.990000000223517</double>
        </property>
        <property name="maximum">
         <double>9999999.990000000223517</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="sourceToHTitle">
        <property name="text">
         <string>终点</string>
        </property>
        <property name="buddy">
         <cstring>sourceToH</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="sourceToH">
        <property name="minimum">
         <double>-9999999.990000000223517</double>
        </property>
        <property name="maximum">
         <double>9999999.990000000223517</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="sourceWidthHTitle">
        <property name="text">
         <string>宽度</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="sourceWidthH">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="gridHTitle">
        <property name="text">
         <string>网格尺寸</string>
        </property>
        <property name="buddy">
         <cstring>gridH</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="gridH">
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="pixelTitleH">
        <property name="text">
         <string>像素数</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="pixelH">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="backlashTitleH">
        <property name="text">
         <string>返程补偿</string>
        </property>
        <property name="buddy">
         <cstring>backlashH</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QDoubleSpinBox" name="backlashH">
        <property name="minimum">
         <double>-99.989999999999995</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="swap">
       <property name="text">
        <string>&lt;= 交换 =&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="../../../Resources/Resources.qrc">
     <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
   <property name="toolTip">
    <string>添加</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="../../resources/icons/Fatcow.qrc">
     <normaloff>:/fatcow/16x16/bullet_edit.png</normaloff>:/fatcow/16x16/bullet_edit.png</iconset>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
   <property name="toolTip">
    <string>编辑</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../../../Resources/Resources.qrc">
     <normaloff>:/icons/minus.png</normaloff>:/icons/minus.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="toolTip">
    <string>删除</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>axisH</tabstop>
  <tabstop>sourceFromH</tabstop>
  <tabstop>sourceToH</tabstop>
  <tabstop>gridH</tabstop>
  <tabstop>backlashH</tabstop>
  <tabstop>scaleH</tabstop>
  <tabstop>axisV</tabstop>
  <tabstop>sourceFromV</tabstop>
  <tabstop>sourceToV</tabstop>
  <tabstop>gridV</tabstop>
  <tabstop>backlashV</tabstop>
  <tabstop>scaleV</tabstop>
  <tabstop>swap</tabstop>
  <tabstop>channels</tabstop>
  <tabstop>channelOffsetList</tabstop>
  <tabstop>addOffset</tabstop>
  <tabstop>editOffset</tabstop>
  <tabstop>deleteOffset</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources/icons/Fatcow.qrc"/>
  <include location="../../../Resources/Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogImageOptions</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>619</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogImageOptions</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>619</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdd</sender>
   <signal>triggered()</signal>
   <receiver>addOffset</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>569</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEdit</sender>
   <signal>triggered()</signal>
   <receiver>editOffset</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>569</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDelete</sender>
   <signal>triggered()</signal>
   <receiver>deleteOffset</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>569</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
